deployments/docker/state-offline-installer/centos/runCompile.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+cd cli
+git pull
+go mod vendor
+sudo /state-tool/bin/state run preprocess
+cd ..
+sudo chown -R as-build:as-build cli
+cd cli/cmd/state-offline-installer/docker
+go build -o /offline-installer-data/offline-installer github.com/ActiveState/cli/cmd/state-offline-installer
```

Comment:
`go mod vendor` should not be part of this, we commit our vendored modules, so by the time you're building this should already be in place.

Also this definitely should be an activestate.yaml script.